lib/src/DateTimePicker/DateTimePickerToolbar.tsx

```diff
@@ -117,25 +123,45 @@ export const DateTimePickerToolbar: React.FC<ToolbarComponentProps> = withDefaul
             <div className={classes.timeContainer}>
               <ToolbarButton
                 tabIndex={-1}
-                variant="h3"
+                variant={timeTypographyText}
                 data-mui-test="hours"
                 onClick={() => setOpenView('hours')}
                 selected={openView === 'hours'}
                 value={date ? formatHours(date) : '--'}
                 typographyClassName={classes.timeTypography}
               />
 
-              <ToolbarText variant="h3" value=":" className={classes.separator} />
+              <ToolbarText variant={timeTypographyText} value=":" className={classes.separator} />
 
               <ToolbarButton
                 tabIndex={-1}
-                variant="h3"
+                variant={timeTypographyText}
                 data-mui-test="minutes"
                 onClick={() => setOpenView('minutes')}
                 selected={openView === 'minutes'}
                 value={date ? utils.format(date, 'minutes') : '--'}
                 typographyClassName={classes.timeTypography}
               />
+
+              {hasSeconds && (
+                <>
+                  <ToolbarText
+                    variant={timeTypographyText}
+                    value=":"
+                    className={classes.separator}
+                  />
+
+                  <ToolbarButton
+                    tabIndex={-1}
+                    variant={timeTypographyText}
+                    data-mui-test="seconds"
+                    onClick={() => setOpenView('seconds')}
+                    selected={openView === 'seconds'}
+                    value={date ? utils.format(date, 'seconds') : '--'}
+                    typographyClassName={classes.timeTypography}
```

Comment:
```suggestion
                    className={classes.timeTypography}
```

Reply:
The Toolbar button has a prop typography class which is passed in to add the class to the typography component in the ToolbarButton, rather than adding the class to the Button directly.

Reply:
Yeah, there is an opportunity here to flatten the DOM structure '